bedevere/bpo.py

```diff
@@ -83,10 +83,74 @@
         await set_status(event, gh)
 
 
+@router.register("issue_comment", action="edited")
+@router.register("issue_comment", action="created")
+@router.register("pull_request", action="edited")
+@router.register("pull_request", action="opened")
+async def hyperlink_bpo_text(event, gh, *args, **kwargs):
+    if "pull_request" in event.data:
+        event_name = "pull_request"
+        body_location = "issue_url"
+    else:
+        event_name = "comment"
+        body_location = "url"
+    if "body" in event.data[event_name] and body_location in event.data[event_name]:
+        body = event.data[event_name]["body"] or ""
+        new_body = create_hyperlink_in_comment_body(body)
+        if new_body != body:
+            body_data = {"body": new_body, "maintainer_can_modify": True}
+            await gh.patch(event.data[event_name][body_location], data=body_data)
+
+
 def create_success_status(found_issue):
     """Create a success status for when an issue number was found in the title."""
     issue_number = found_issue.group("issue")
     url = f"https://bugs.python.org/issue{issue_number}"
     return util.create_status(STATUS_CONTEXT, util.StatusState.SUCCESS,
                               description=f"Issue number {issue_number} found",
                               target_url=url)
+
+
+def check_hyperlink(match):
+    """The span checking of regex matches takes care of cases like bpo-123 [bpo-123]…"""
+    issue = match.group('issue')
```

Comment:
Please use double quotes for your changes.
I know we haven't been consistent in the past, but since this is new code, might as well be more consistent about it. :)